src/static/riot/submissions/submission_management.tag

```diff
@@ -0,0 +1,266 @@
+<submission_management>
+    <!-- HTML -->
+    <h1>Submission Management</h1>
+    <div class="ui divider"></div>
+    <div class="stackable ui menu">
+        <div class="item">
+            <div class="ui icon input">
+                <input type="text" placeholder="Search Users...">
+                <i class="user icon"></i>
+            </div>
+        </div>
+        <div class="ui right pushed search item">
+            <div class="ui icon input">
+                <input class="prompt" type="text" placeholder="Search Competitions...">
+                <i class="search icon"></i>
+            </div>
+            <div class="results"></div>
+        </div>
+    </div>
+    <div id="horizButtons" class="ui horizontal icon buttons">
```

Comment:
I think we should get in the habit of naming all IDs [snake_case](https://en.wikipedia.org/wiki/Snake_case) just to keep a consistent style

Reply:
changed id case to snake_case